ambariclient/base.py

```diff
@@ -396,7 +397,11 @@ class Model(object):
     min_version = OLDEST_SUPPORTED_VERSION
 
     def __init__(self, parent, data=None):
-        self._data = data if data is not None else {}
+        if data is None:
+            data = {}
+
+        self._data = dict((key, value) for key, value in six.iteritems(data)
```

Comment:
Well, we're the only one using it and we're pinned to the pypi version, so it won't automatically get used anywhere.  I won't push to pypi until I've fully tested it.